package.json

```diff
@@ -1,6 +1,16 @@
 {
   "name": "strong-mesh-client",
   "version": "2.0.1",
+  "keywords": [
+    "strongloop",
+    "mesh"
+  ],
```

Comment:
We don't need keywords, because no one needs to search for this package and find it.